test/e2e/specs/example.spec.js

```diff
@@ -13,25 +13,26 @@ test.describe("Example Spec", () => {
         await screenshot(testInfo, page);
     });
 
-    test("change display timezone", async ({ page }, testInfo) => {
```

Comment:
Is removing the Timezone testcase intentional?

Reply:
Yes. I discovered my example spec that was intended to also test resetting the database was actually flawed. The database reset wasn't occurring (discovered while writing this Status Page spec) because the test server isn't started in a dev environment to expose the e2e routes. The example test was succeeding when it wasn't supposed to because (if I determined correctly) localStorage saving the timezone. So I changed the test to something that isn't prone to such issue.